entity-service-api/src/main/proto/org/hypertrace/entity/query/service/v1/entity_query_service.proto

```diff
@@ -18,4 +18,6 @@ service EntityQueryService {
   }
   rpc total (TotalEntitiesRequest) returns (TotalEntitiesResponse) {
   }
+  rpc bulkUpdateEntityArrayAttribute (BulkEntityArrayAttributeUpdateRequest) returns (stream ResultSetChunk) {
```

Comment:
we don't need to (or want to) stream results. We've stopped doing that for new rpcs - can we give it a plain unary response like `BulkEntityArrayAttributeUpdateResponse`